controller/api/models.py

```diff
@@ -1069,3 +1134,5 @@ def create_auth_token(sender, instance=None, created=False, **kwargs):
     post_delete.connect(_etcd_purge_app, sender=App, dispatch_uid='api.models')
     post_save.connect(_etcd_publish_cert, sender=Certificate, dispatch_uid='api.models')
     post_delete.connect(_etcd_purge_cert, sender=Certificate, dispatch_uid='api.models')
+    post_save.connect(_etcd_publish_config, sender=Config, dispatch_uid='api.models')
```

Comment:
@bacongobbler We should also update http://docs.deis.io/en/latest/managing_deis/upgrading-deis/#triggering-an-upgrade as part of this PR since we'll now need to republish Config objects after an upgrade.